mods/seasonal/moves.js

```diff
@@ -66,6 +66,34 @@ exports.BattleMovedex = {
 		target: "normal",
 		type: "Dark",
 	},
+	// antemortem
+	postmortem: {
+		acc: 85,
+		basePower: 110,
+		category: "Special",
+		shortDesc: "Makes contact. 50% boost Spa, Spe.",
+		id: "postmortem",
+		isNonstandard: true,
+		name: "Postmortem",
+		pp: 10,
```

Comment:
PP is supposed to be 10, so we need to add `noPPBoosts: true,`

Reply:
Done but didn't show up here for some reason